.release-please-manifest.json

```diff
@@ -19,7 +19,6 @@
   "apps/raster": "1.0.5",
   "apps/eppo": "1.0.5",
   "apps/livestory": "1.0.2",
-  "apps/lottie-viewer": "1.0.0",
```

Comment:
We shouldn't have manually set this in here to v1. release please config triggers any new app to be released at version 1 and updates this/the app's package json automatically, I think with the release main PR.